src/main/webapp/src/app/datepicker/datepicker.component.ts

```diff
@@ -50,4 +56,20 @@ export class DatepickerComponent implements OnInit {
     }
     return formattedDate;
   }
+
+  /**
+   * Return the selected date; the time is by default the start of the day
+   */
+  get selectedDate(): string {
+    return this.selectedDateIso;
+  }
+
+  /**
+   * Return the selected date with the time being the end of the day
```

Comment:
Curious - why the end of the day?

Reply:
When listing events, I needed to filter the events from the start of the day till the end of the day so only the events with the selected date appear.

Reply:
So all day events will always be at the end?